druid-shell/src/backend/wayland/window.rs

```diff
@@ -106,10 +106,25 @@ impl WindowHandle {
         tracing::warn!("resizable is unimplemented on wayland");
     }
 
+    pub fn is_resizable(&self) -> bool {
+        tracing::warn!("is_resizable is unimplemented on wayland");
+        true
+    }
+
+    pub fn is_transparent(&self) -> bool {
+        tracing::warn!("is_transparent is unimplemented on wayland");
+        false
```

Comment:
warning is unnecessary for transparent. it'll really is unnecessary for wayland. wayland allows transparency by default. depending on the intent of this method we may wish to return true.

Reply:
Didn't know that! Would change to true always.